### PR TITLE
D8CORE-000: Fixes broken filtering logic on view field.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Makes the display filtering with the alter hook actually work. 

# Needed By (Date)
- Today

# Urgency
- Hight

# Steps to Test

1. Check out this branch
2. Check out branch https://github.com/SU-SWS/react_paragraphs/pull/45
3. Clear all caches
4. Create a new basic page
5. Add news view paragraph and see only two options (Card + List)

# Affected Projects or Products
- D8CORE

### CODE REVIEW

File: stanford_news.module [hunk 1]
```diff
@@ -179,7 +179,7 @@ function stanford_news_field_widget_form_alter(&$element, FormStateInterface $fo
 /**
  * Implements hook_react_paragraphs_getfieldinfo_post_alter().
  */
-function stanford_news_react_paragraphs_getfieldinfo_post_alter($field_element, FieldConfigInterface $field_config, &$info) {
+function stanford_news_react_paragraphs_getfieldinfo_post_alter(array &$info, array $field_element, FieldConfig $field_config) {
```
Comment (pookmish) [inline stanford_news.module:182]: hook name changed too

Comment (pookmish) [reply stanford_news.module:182]:
```suggestion
function stanford_news_react_paragraphs_form_field_data_alter(array &$info, array $field_element, FieldConfigInterface $field_config) {
```

Comment (pookmish) [reply stanford_news.module]: _NEVERMIND_

Comment (imonroe) [inline stanford_news.module]: dat function name tho :squint:

File: stanford_news.module [hunk 2]
```diff
@@ -177,9 +177,9 @@ function stanford_news_field_widget_form_alter(&$element, FormStateInterface $fo
 }
 
 /**
- * Implements hook_react_paragraphs_getfieldinfo_post_alter().
+ * Implements hook_react_paragraphs_form_field_data_alter().
  */
-function stanford_news_react_paragraphs_getfieldinfo_post_alter($field_element, FieldConfigInterface $field_config, &$info) {
+function stanford_news_react_paragraphs_form_field_data_alter(array &$info, array $field_element, FieldConfig $field_config) {
```
Comment (pookmish) [inline stanford_news.module:182]:
```suggestion
function stanford_news_react_paragraphs_form_field_data_alter(array &$info, array $field_element, FieldConfigInterface $field_config) {
```

File: stanford_news.module (hunk 1 above)
Comment (sherakama) [reply stanford_news.module]: of course it did

Comment (sherakama) [reply stanford_news.module]: Mike "fixed" it.

**Pull request merged**